iml-manager-cli/src/main.rs

```diff
@@ -172,20 +191,60 @@ fn main() {
                             purge_duration: pd
                         }),
                     )
+                    .and_then(|resp| iml_manager_client::concat_body(resp))
+                    .map(|(resp, chunk)| {
+                        let status = resp.status();
+                        match status.as_u16() {
```

Comment:
Instead of matching over status code ranges, this would be more clear if you used the methods exposed on `StatusCode` with if ... else. https://docs.rs/reqwest/0.9.18/reqwest/struct.StatusCode.html#method.is_success, https://docs.rs/reqwest/0.9.18/reqwest/struct.StatusCode.html#method.is_client_error

Reply:
sure